staging/src/k8s.io/apimachinery/pkg/util/wait/backoff.go

```diff
@@ -200,19 +204,44 @@ func NonSlidingUntilWithContext(ctx context.Context, f func(context.Context), pe
 //
 // Close stopCh to stop. f may not be invoked if stop channel is already
 // closed. Pass NeverStop to if you don't want it stop.
+//
+// Contextual logging: JitterUntilWithContext should be used instead of JitterUntil in code which supports contextual logging.
 func JitterUntil(f func(), period time.Duration, jitterFactor float64, sliding bool, stopCh <-chan struct{}) {
 	BackoffUntil(f, NewJitteredBackoffManager(period, jitterFactor, &clock.RealClock{}), sliding, stopCh)
 }
 
+// JitterUntilWithContext loops until context is done, running f every period.
+//
+// If jitterFactor is positive, the period is jittered before every run of f.
+// If jitterFactor is not positive, the period is unchanged and not jittered.
+//
+// If sliding is true, the period is computed after f runs. If it is false then
+// period includes the runtime for f.
+//
+// Cancel context to stop. f may not be invoked if context is already expired.
```

Comment:
nit: this uses `expired` but `done` is used in most other places.  Might make sense to use `done` here as well but this is minor and I can understand either preference

Reply:
This is the original documentation. I just moved it around as part of refactoring:

https://github.com/kubernetes/kubernetes/blob/v1.32.0/staging/src/k8s.io/apimachinery/pkg/util/wait/backoff.go#L249-L260

I agree that "done" makes more sense, so let's clean that up in this PR... [done](https://github.com/kubernetes/kubernetes/compare/adc2c0137bb907c0c1f0d057022cf694b08e0bda..e681a790584ada4c0a7813f32698fc3ff0784305). Please re-add the LGTM.